docs/about.md

```diff
@@ -1,6 +1,6 @@
 # About Contour and Envoy
 
-At the highest level Contour is a proxy that fulfills the Envoy Cluster Discovery Service (CDS) API by translating Kubernetes Ingress objects into JSON configuration fragments.
+At the highest level Contour is a proxy that fulfills the Envoy Cluster Discovery Service (CDS) API by translating Kubernetes Ingress objects and IngressRoute objects into JSON configuration fragments.
```

Comment:
Now that we no longer support the JSON API, I think we can update "JSON configuration fragments" as well. Maybe something like: "into Envoy configuration objects" ?

Reply:
:+1: